src/Raven.Studio/wwwroot/App/views/database/advanced/tombstonesState.html

```diff
@@ -48,13 +47,11 @@
         </div>
         <div class="flex-grow scroll">
             <div class="panel padding padding-sm">
-                <h3>Per Collection State</h3>
+                <h3>Per Collection - Max Etags that can be deleted</h3>
                 <div style="position: relative; height: 50vh">
                     <virtual-grid class="resizable" params="controller: collectionsStateController"></virtual-grid>    
                 </div>
-                
-                <h3>Per Subscription State</h3>
-                
+                <h3>Per Process - Max Etag that can be deleted</h3>
```

Comment:
why we don't mention word subscription here?

Reply:
As discussed, refactored: `Process => Task `